docs/explanation.rst

```diff
@@ -183,6 +183,28 @@ not (this is a particular concern when working with X-ray or gamma-ray data).
 .. _TAI: https://www.bipm.org/en/bipm-services/timescales/tai.html
 .. _wiki: https://github.com/nanograv/PINT/wiki/Clock-Corrections-and-Timescales-in-PINT
 
+Dispersion Measure
+------------------
+Radio waves emitted by the pulsar experience dispersion as they travel through the ionized 
+interstellar medium (ISM). The time delay due to the interstellar dispersion is given by
+:math:`\frac{K\times DM}{\nu^2}`, where :math:`\nu` is the frequency of the radio signal.
+The dominant source of this dispersion is the presence of free electrons in the ISM, and 
+to a first approximation, the DM can be interpreted as the electron column density along the 
+line of sight to the pulsar. :math:`K` is known as the DM constant, and should be equal to 
+:math:`\frac{e^2}{8 \pi ^2 c \epsilon _0 m_e} \approx 1.3445365918(9)E-07\; m^2/s` based
```

Comment:
Could you do `E-07` -< `\times 10^{-7}`?  The former doesn't format well.  And this is done below also.

Reply:
Done.